lib/mix/lib/mix/tasks/new.ex

```diff
@@ -445,8 +445,8 @@ defmodule Mix.Tasks.New do
     use ExUnit.Case
     doctest <%= @mod %>
 
-    test "the truth" do
-      assert 1 + 1 == 2
+    test "salutates the world" do
+      assert <%= @mod %>.hello == :world
```

Comment:
@uesteibar it is a regex, so you need to escape the parens inside the regex. Or, even better, don't use a regex at all:

    file =~ "assert HelloWorld.hello() == :world"